templates-assets/templates/NotificationAAR_RADDalt/i18n/fr.json

```diff
@@ -32,5 +33,6 @@
   "delegate_list_2":"sa pièce d'identité;",
   "delegate_list_3":"son numéro d’identification fiscale;",
   "delegate_list_4":"une copie ou l'original du code fiscal et une pièce d'identité de la personne délégante",
-  "delegate_list_5":"un documento che attesti il tuo ruolo di legale rappresentante."
+  "delegate_list_5":"un documento che attesti il tuo ruolo di legale rappresentante.",
```

Comment:
here the sentence is "un documento che attesti il ruolo di legale rappresentante del delegante."